src/controllers/privateMessage.controller/getLatestPrivateMessage/index.js

```diff
@@ -23,8 +23,23 @@ const getLatestPrivateMessage = catchAsync(async (req, res) => {
   };
 
   const response1 = await Hasura(getPrivateMessages, variables);
+  const messageDocs = response1.result.data.private_messages;
 
-  const decryptedMessages = await decryptMessages(response1.result.data.private_messages);
+  let decryptedMessages = [];
+
+  for (let i = 0; i < messageDocs.length; i++) {
```

Comment:
Please use iterators like map etc. Its never good to have for loops in production code, only use if necessary.